client/core/core.go

```diff
@@ -1169,6 +1171,11 @@ func New(cfg *Config) (*Core, error) {
 		return nil, err
 	}
 
+	seedGenerationTime, err := boltDB.SeedGenerationTime()
+	if err != nil && !errors.Is(err, db.ErrNoSeedGenTime) {
+		return nil, err
+	}
```

Comment:
Instead of proceeding zero if `err` is `db.ErrNoSeedGenTime`, why don't we hard code the same value as used with btc since that's basically the earliest for an hd key.

Reply:
I'm actually checking if seedgentime = 0 over here:
https://github.com/martonp/dcrdex/blob/3d7963aa63a89df3551414e621c1ce0f3171234a/client/webserver/site/src/js/forms.js#L122

If the time the seed was generated is known, the default is the current date, so basically no scanning needs to be done. Otherwise, the default date that the wallet provides is used. The way it is currently used, `core.seedgentime` could just be a boolean that tells whether or not the current installation generated the seed.